main.py

```
@@ -6,22 +6,32 @@
 from fastapi import FastAPI
 from fastapi.exceptions import RequestValidationError
 from api.frame.frame_routes import router as frame_router
+from api.diagnosis.diagnosis_routes import router as diagnosis_router
```

Ruff (0.11.9)

Module level import not at top of file

(E402)